src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
Yes, either way remaining stress modes will be in effect.

To just run with one stress mode enabled, we would have to use the following settings (I know it is a bit clumsy)

complus_jitstress=value
complus_JitStressModeNames=STRESS_RANDOM_INLINE
complus_JitStressModeNamesNot="delimited list of all stress mode names listed in compiler.h other than RANDOM_INLINE"

To make it easier, we can think of adding a new config option (say complus_JitStressModeNamesOnly) to exercise only the stress modes specified with complus_JitStressModeNames setting.